apps/asap-server/test/handlers/groups/fetch.fixtures.ts

```diff
@@ -0,0 +1,292 @@
+            tools: [
+              {
+                url: 'https://example.com/secure-comms',
+                name: 'Slack',
+                description: 'Secret Comms',
+              },
+            ],
+            calendars: [
+              {
+                id: 'calendar-id-1',
+                created: '2020-12-11T14:33:50Z',
+                lastModified: '2020-12-11T15:06:26Z',
+                flatData: {
+                  color: '#B1365F',
+                  id: 'hub@asap.science',
+                  name: 'ASAP Hub',
+                },
+              },
+            ],
+            teams: [
+              {
+                id: 'team-id-1',
+                created: '2020-12-11T14:33:50Z',
+                lastModified: '2020-12-11T15:06:26Z',
+                flatData: {
+                  applicationNumber: 'ASAP-000592',
+                  displayName: 'Lee, M',
+                  outputs: [
+                    {
+                      id: 'output-id-1',
+                      created: '2020-12-11T14:33:18Z',
+                      flatData: {
+                        link: null,
+                        publishDate: null,
+                        shortText: null,
+                        title: 'Proposal',
+                        type: 'Proposal',
+                      },
+                    },
+                  ],
+                  projectSummary: null,
+                  projectTitle:
+                    'Senescence in Parkinson’s disease and related disorders',
+                  skills: [],
+                  proposal: [
+                    {
+                      id: 'output-id-1',
+                    },
+                  ],
+                  tools: [],
+                },
+              },
+            ],
+            leaders: [
+              {
+                role: 'Lead PI - Chair',
+                user: [
+                  {
+                    id: 'user-id-1',
+                    created: '2020-12-11T14:33:50Z',
+                    lastModified: '2020-12-11T15:06:26Z',
+                    flatData: {
+                      avatar: [
+                        {
+                          id: 'asset-id-1',
+                        },
+                      ],
+                      social: [
+                        {
+                          github: 'fampinheiro',
+                          googleScholar: null,
+                          linkedIn: 'fampinheiro',
+                          researcherId: null,
+                          researchGate: null,
+                          twitter: 'fampinheiro',
+                        },
+                      ],
+                      biography:
+                        'Filipe is an Engineering Manager who works closely with our clients to ensure process and technical standards improvements. He previously worked at startups and multinational companies, leading tech projects in mobile and web apps, before bringing this gathered experience to YLD.',
```

Comment:
The anonymised data is great :slightly_smiling_face:.

Reply:
What Filipe? 🙄 I know no Filipe